MatrixSDK/Background/MXBackgroundStore.swift

```diff
@@ -83,45 +58,177 @@ class MXBackgroundStore: MXMemoryStore {
     }
     
     //  This store should act like as a permanent one
-    override var isPermanent: Bool {
+    var isPermanent: Bool {
         return true
```

Comment:
```suggestion
        return false
```

I think we may stop acting like a permanent store, as we don't use a session anymore.